bulksendemail.js

```diff
@@ -0,0 +1,231 @@
+/*
+ * @version    SVN:<SVN_ID>
+ * @package    TJsendemail
+ * @author     Techjoomla <extensions@techjoomla.com>
+ * @copyright  Copyright (c) 2009-2018 TechJoomla. All rights reserved
+ * @license    GNU General Public License version 2, or later
+ */
+
+
+var tjutilitysendemail = {
+    initialize: function(tableContainer) {
+		var isSendEmail = jQuery('body').find('.td-sendemail').length;
+		var isCheckboxes = jQuery('body').find('input[name="cid[]"]').length;
+
+		if (isSendEmail)
+		{
+			// If on list view already checkbox are present then avoid the add column function
+			if (!isCheckboxes)
+			{
+				this.addColumn(tableContainer);
+			}
+
+			this.btnSendEmail();
+		}
+    },
+    addColumn: function(tblId)
+	{
+		var tr = document.getElementById(tblId).tHead.children[0];
+		tr.insertCell(0).outerHTML = '<th><input type="checkbox" name="checkall-toggle" value="" class="hasTooltip" title="Check All Items" onclick="Joomla.checkAll(this)"></th>'
+
+		var tblBodyObj = document.getElementById(tblId).tBodies[0];
+		for (var i=0; i<tblBodyObj.rows.length; i++) {
+			var newCell = tblBodyObj.rows[i].insertCell(0);
+			newCell.innerHTML = '<input type="checkbox" id="cb0" name="cid[]" value="' + i + '" onclick="Joomla.isChecked(this.checked);">'
+		}
+	},
+	btnSendEmail: function () {
+		try {
+			var alertMessage = 'alert("Please select recods");';
+			let btnHtml = '<div class="btn-wrapper" id="tj-sendemail" style="float: right;">';
+					btnHtml += '<button type="button" class="btn btn-primary" id="email-queue-column" data-toggle="modal" data-target="#builkEmailModal" onclick="tjutilitysendemail.openEmailPopup();">';
+						btnHtml += Joomla.JText._('PLG_SYSTEM_SENDEMAIL_BTN') ;
+					btnHtml += '</button>';
+				btnHtml += '</div>';
+
+			jQuery('#toolbar').append(btnHtml);
+
+		}
+		catch (err) {
+			/*console.log(err.message);*/
+		}
+	},
+	openEmailPopup: function () {
+		try {
+
+			if (document.adminForm.boxchecked.value==0)
+			{
+				alert(Joomla.JText._('PLG_SYSTEM_SENDEMAIL_SELECT_CHECKBOX'));
+				return false;
+			}
+
+			// Remove below line removeclass it is temp added
+			jQuery("div").find('.tjlms-wrapper').removeClass("tjBs3");
+
+			let modelEmail = '<div id="builkEmailModal" class="emailModal modal fade" role="dialog">';
```

Comment:
Check support for `let` across browsers.

Also - is there a better way to declare this HTML ?